### PR TITLE
Streamline Settings > Advanced > Features section

### LINKED ISSUE #1198: Simplify settings under 'Advanced > Features'
### Description

The WooCommerce settings available under **Settings > Advanced** give users the ability to enable/disable experimental WooCommerce features, and features that are not yet available in Core. Among other things, users can:

- Disable WooCommerce Analytics
- Enable HPOS

Woo Express is meant to provide an experience that is optimized for merchants, and the WP.com hosting infrastructure. Giving users full control to experiment with HPOS, or disable Core Analytics does not make sense in an environment meant to simplify decisions for users.

That said, we should make it easy for users to un-do Woo Express features and customizations that make their experience diverge significantly from what some might already expect from WooCommerce.

In line with these ideas, the objective of this task is to:

- Hide all existing settings under **Settings > Advanced > Features**.
- Introduce a new setting to allow users to revert to the default Woo navigation.

### PR DESCRIPTION
**Pull request by manospsyx**
### Changes proposed in this Pull Request:

Hide all existing settings under **Settings > Advanced > Features**. Introduce a setting to let users revert back to the self-hosted Woo menu.

Closes #1198 .
- #1198 

### How to test the changes in this Pull Request:

- View the **Settings > Advanced > Features** page. 
- Confirm that this shows up:

Screenshot 2023-06-23 at 3 47 28 PM

- Confirm that the Advanced tab has moved to the end.

### CODE REVIEW

**Comment by manospsyx** [comment]
@xristos3490 got any tips re `wp_unslash` and PHPCS?

File: includes/class-wc-calypso-bridge-woocommerce-admin-features.php [hunk 1]
```diff
@@ -39,7 +40,7 @@ public static function get_instance() {
 	 */
 	public function __construct() {
 
-		// Only in Ecommerce.
+		// Only in Woo Express plans.
```
Comment (xristos3490) [inline includes/class-wc-calypso-bridge-woocommerce-admin-features.php:43]: This isn't only for Woo Express. It also includes the wpcom's ecommerce plan.

**Review by xristos3490 — COMMENTED** [review]
Looks good! I've added a few comments!

File: includes/class-wc-calypso-bridge-woocommerce-admin-features.php [hunk 2]
```diff
@@ -56,11 +56,26 @@ public function initialize() {
...
+		add_filter( 'woocommerce_get_settings_advanced', array( $this, 'filter_woocommerce_settings_features' ), 1000, 2 );
+		add_filter( 'woocommerce_settings_tabs_array', array( $this, 'filter_woocommerce_settings_pages_order' ), 1000 );
+
+		/*
+		 * Refresh the page to get the menu to reload when saving the Settings under 'Advanced > Features'.
+		 */
+		add_action( 'woocommerce_settings_saved', function() {
+			global $current_section, $current_tab;
+			if ( 'advanced' === $current_tab and 'features' === $current_section ) {
+				wp_redirect( $_SERVER['REQUEST_URI'] );
```
[the remark this answers is not in the document]
Comment (xristos3490) [reply includes/class-wc-calypso-bridge-woocommerce-admin-features.php]: Oh, I see you already changed this one. Although `sanitize_url` is better, the `esc_url_raw` is also working fine.

**Review by PanosSynetos — APPROVED** [review]
Pair reviewed with @manospsyx  and manually tested - We cool, ready to ship!

**Pull request merged**